station_data.py

```diff
@@ -4,8 +4,21 @@
 from StringIO import StringIO
 from numpy import genfromtxt
 import ast
+import re
```

Comment:
I tried a solution using requests but get ImportError: No module named requests. How would I fix that?